### DISCUSSION TITLE
General Nginx configuration for EJBCA

### DISCUSSION THREAD

**Discussion by eryabies** [Keyfactor/ejbca-ce#363] [General]
Hi!

Is there an official front end config file for Nginx like there is for Apache? (https://doc.primekey.com/ejbca/ejbca-integration/integrating-with-third-party-applications/setting-up-an-apache-web-server-as-a-proxy)

**Comment by primetomas** [comment]
I don't have an nginx config at hand, but it should match this apache config.
https://github.com/Keyfactor/keyfactorcommunity/tree/main/deployment-examples/docker-engine/ejbca-ce-three-level-architecture

**Comment by svenska-primekey** [comment]
If you do get a working nginx config then please feel free to share a link here.

**Comment by aloeenmae** [comment]
I'm using this kind of nginx conf. It is still in beta stage, but working.
[https://github.com/aloeenmae/ejbca-ce/blob/6c16283f3c5d89c0ca728bc69d245d02413cd239/nginx/conf/frontend.conf](url)
